webpack.config.js

```diff
@@ -22,7 +22,7 @@ module.exports = {
   development:{
    browser: {
       name     : 'browser',
-      devtool  : '#inline-source-map',
+      // devtool  : '#inline-source-map',
```

Comment:
Maybe `cheap-module-eval-source-map	` for development

http://webpack.github.io/docs/configuration.html#devtool